pkg/config/resource.go

```diff
@@ -328,11 +329,20 @@ func (t *Tagger) Initialize(ctx context.Context, mg xpresource.Managed) error {
 	return nil
 }
 
+const externalResourceTagKeyExternalName = "crossplane-external-name"
+
+func getExternalTags(mg xpresource.Managed) map[string]string {
+	externalTags := xpresource.GetExternalTags(mg)
```

Comment:
This was the only call to `xpresource.GetExternalTags` I could find in the code, so I'm assuming this is the best and only place to provide this additional tag. If there are other places I'm missing, please let me know and I'll adjust.